wiki/Debugging.md

```diff
@@ -20,6 +20,62 @@
     while on bootsplash
 
 ### Stuck at bootanimation/splash (or need regular logs but ADB is dead)
+1) Reboot into recovery.

```

Comment:
I'd suggest calling this section something along the lines of "Fixing ADB not authorized"

Reply:
what about "Manually authorize ADB to capture logs during boot"

+ won't it create confusion with the heading available below https://github.com/omansh-krishn/opendroid-docs/blob/master/wiki/Debugging.md#adb-not-authorized

Reply:
What about something like  "Collecting logs from ADB by disabling authorization", since we are trying to make adb run in non-secure mode instead of "authorizing" it

Reply:
it's more like bypassing instead of authorizing

I just am not able to come up with a term for what's being bypassed 😢

Reply:
"Temporarily bypassing" maybe?

Reply:
nah, it becomes permanent after you push back the build.prop,

Reply:
I finally came up with this
"Override ADB Authentication to capture logs during boot".